.gitlab/test_ep.sh

```
@@ -0,0 +1,105 @@
+    echo "---- elastic: plan=$(basename "$plan_file") flags=[$extra_flags] ----"
+    (
+        unset NIXL_ETCD_ENDPOINTS NIXL_ETCD_PEER_URLS NIXL_ETCD_NAMESPACE
+        unset UCX_NET_DEVICES  # let UCX auto-select GPU-capable transport
+        # Force NVLink-only transports.
+        if [[ "$extra_flags" != *--disable-ll-nvlink* ]]; then
+            export UCX_TLS=cuda_copy,cuda_ipc,sm,self
+        fi
+        PYTHONPATH="${NIXL_BUILD_DIR}/${EP_SRC_DIR}:${EP_SRC_DIR}/tests:${EP_SRC_DIR}/tests/elastic${PYTHONPATH:+:$PYTHONPATH}" \
+        timeout 300 python3 ${EP_SRC_DIR}/tests/elastic/elastic.py \
+            --plan "$plan_file" \
+            --num-processes 4 --num-topk 4 --validate-phase-failures $extra_flags
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)